### PR TITLE
Labels field documentation

### PR DESCRIPTION
**Pull request by rob8714**
- add documentation about labels field
- add reference to labels in commonLabels
- re-order kustomization section alphabetically

Information about the labels field added from release notes: https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv4.1.0

First commit has changes, second commit regenerates html

Netlify link: https://eager-heisenberg-7774e6.netlify.app/

### CODE REVIEW

File: site/content/en/references/kustomize/kustomization/commonlabels/_index.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ description: >
 Add labels and selectors to all resources.  If the label key already is present on the resource,
 the value will be overridden.
 
+An alternative to this field is the `labels` field, which allows adding labels without also automatically
```
Comment (KnVerey) [inline site/content/en/references/kustomize/kustomization/commonlabels/_index.md:13]: Please link to your new page here

**Review by KnVerey — COMMENTED** [review]
Thanks for the PR! Can you please rebase?

> re-order kustomization section alphabetically

I see in the preview that there are still some issues towards the end of the list:

image

**Comment by rob8714** [comment]
@KnVerey thanks, going to regenerate the html soon, already did the rebase and fixed what you highlighted

**Comment by rob8714** [comment]
@KnVerey changes done and html regenerated

**Pull request merged**